src/components/facecard/facCard.js

```diff
@@ -1,38 +1,74 @@
-import React, { useState } from "react";
-import './faceCard.css';
-import User from '../../images/user.png';
-import Email from '../../images/envelope-open.png';
-import Calendar from '../../images/calendar.png';
-import Address from '../../images/map.png';
-import Phone from '../../images/call-incoming.png';
-import Password from '../../images/lock.png';
+import React, { useState, useEffect } from "react";
+import { getRandomUser } from "./actions/randomUserActions";
+import "./faceCard.css";
+import User from "../../images/user.png";
+import Email from "../../images/envelope-open.png";
+import Calendar from "../../images/calendar.png";
+import Address from "../../images/map.png";
+import Phone from "../../images/call-incoming.png";
+import Password from "../../images/lock.png";
 
 const FaceCard = () => {
-
   const [randomUser, setRandomUser] = useState();
+  const [userInfo, setUserInfo] = useState();
+  const [infoPretext, setInfoPretext] = useState('My name is');
+
+
+  useEffect(() => {
+    getRandomUser().then((user) => {
+      setRandomUser(user.results[0]);
```

Comment:
check to see if you can call the rest of the setStates outside the randomUser call